makefiles/Makefile.win.mk

```diff
@@ -21,6 +21,10 @@ STATIC_PRE_LIB = $(OR_ROOT)lib\\
 STATIC_POST_LIB = .lib
 O=obj
 E=.exe
+L=.lib
```

Comment:
@lperron One thought occurred to me, is the *csharp* invoked by the *unix* make files? If so, no problem.